- Change default theme to Xfce.
- Change default icon theme to gnome.
- Enable Antialias and Hinting by default.

--- xfsettingsd/xsettings.xml.orig	2012-04-19 18:13:36.036629829 +0200
+++ xfsettingsd/xsettings.xml	2012-04-19 18:15:08.410392293 +0200
@@ -6,8 +6,8 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <channel name="xsettings" version="1.0">
   <property name="Net" type="empty">
-    <property name="ThemeName" type="empty"/>
-    <property name="IconThemeName" type="empty"/>
+    <property name="ThemeName" type="Xfce"/>
+    <property name="IconThemeName" type="gnome"/>
     <property name="DoubleClickTime" type="int" value="250"/>
     <property name="DoubleClickDistance" type="int" value="5"/>
     <property name="DndDragThreshold" type="int" value="8"/>
@@ -19,9 +19,9 @@
   </property>
   <property name="Xft" type="empty">
     <property name="DPI" type="empty"/>
-    <property name="Antialias" type="int" value="-1"/>
-    <property name="Hinting" type="int" value="-1"/>
-    <property name="HintStyle" type="string" value="hintnone"/>
+    <property name="Antialias" type="int" value="1"/>
+    <property name="Hinting" type="int" value="-1"/>
+    <property name="HintStyle" type="string" value="hintfull"/>
     <property name="RGBA" type="string" value="none"/>
     <!-- <property name="Lcdfilter" type="string" value="none"/> -->
   </property>
